docs/guides/documentdb/autoscaler/storage/index.md

```
@@ -0,0 +1,350 @@
+At first, verify that your cluster has a storage class that supports volume expansion.
+
+```bash
+$ kubectl get storageclass
+NAME                   PROVISIONER             RECLAIMPOLICY   VOLUMEBINDINGMODE      ALLOWVOLUMEEXPANSION   AGE
+local-path (default)   rancher.io/local-path   Delete          WaitForFirstConsumer   false                  22d
+longhorn               driver.longhorn.io      Delete          Immediate              true                   18d
+```
+
+We can see the `longhorn` storage class has `ALLOWVOLUMEEXPANSION` set to `true`, and it supports online volume expansion, so we will use it. You can install longhorn from [here](https://longhorn.io/docs/).
+
+#### Deploy DocumentDB Cluster
```

markdownlint-cli2 (0.22.1)

[warning] Heading levels should only increment by one level at a time
Expected: h3; Actual: h4

(MD001, heading-increment)